Constants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Rsk.Saml.Samples;
+
+public static class Constants
```

Comment:
Suggest calling the class and filename LicenseKey then it's very obvious whats inside it